quiche/src/lib.rs

```diff
@@ -495,6 +495,9 @@ const DEFAULT_INITIAL_CONGESTION_WINDOW_PACKETS: usize = 10;
 // The maximum data offset that can be stored in a crypto stream.
 const MAX_CRYPTO_STREAM_OFFSET: u64 = 1 << 16;
 
+// The send capacity factor.
+const TX_CAP_FACTOR: usize = 1;
```

Comment:
It is strange that this is an integer.

Reply:
Changed to `f64` though TBH I think we'd still want something like 2, but at least now we can experiment with more values.